### PR TITLE
[#336] make LRA service to wait for nested LRAs in the TCK nestedActivity tests

### LINKED ISSUE #336: TCK nestedActivity test uses the LRATestService to wait for top level LRA which was not finished yet
The `*nestedActivity` tests (tests which use method `multiLevelNestedActivity`) use the `LRATestService` to wait for finishing the LRAs (getting the completion callback being called). But the service is instrumented to wait for the top-level LRA which is not finished at that time. Only nested LRAs should be verified as being finished.

See the call which waits for top-level LRA at https://github.com/eclipse/microprofile-lra/blob/1.0-M1/tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java#L620

### PR DESCRIPTION
**Pull request by ochaloup**
fixes #336

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java [hunk 1]
```diff
@@ -604,7 +604,10 @@ private void multiLevelNestedActivity(CompletionType how, int nestedCnt) throws
         assertFalse("multiLevelNestedActivity: top level LRA should be active (path called " + resourcePath.getUri() + ")",
                 lraTestService.isLRAFinished(URI.create(lraArray[0])));
 
-        lraTestService.waitForCallbacks(lra);
+        // starting at index 1 as LRAResource#multiLevelNestedActivity returns the top-level LRA as the first argument which was not finished yet
+        IntStream.range(1, uris.length).forEach(i -> {
```
Comment (xstefank) [inline tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java:608]: looks good, but maybe we can `.parallel()` this?

**Review by xstefank — APPROVED** [review]

File: tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java (hunk 1 above)
Comment (ochaloup) [reply tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java]: Ok, I added the `.parallel()` here.

**Comment by mmusgrov** [comment]
@ochaloup I think we can merge test issues without needing to reach out to the broader community. If you agree then this one can be merged.

**Comment by ochaloup** [comment]
@mmusgrov yes, I agree.

**Comment by xstefank** [comment]
agreed, merging. Thanks.

**Pull request merged**